components/admin_console/admin_definition.jsx

```diff
@@ -3555,23 +3547,27 @@ export default {
                 component: MessageExportSettings,
             },
         },
-        compliance: {
-            url: 'compliance/settings',
-            title: t('admin.sidebar.compliance'),
-            title_default: 'Compliance Settings',
-            isHidden: it.isnt(it.licensedForFeature('Compliance')),
+        audits: {
+            url: 'compliance/monitoring',
+            title: t('admin.sidebar.complianceMonitoring'),
+            title_default: 'Compliance Monitoring',
+            isHidden: it.isnt(it.licensed),
+            searchableStrings: [
+                'admin.audits.title',
```

Comment:
'admin.audits.title' was changed in favor of ['admin.complianceMonitoring.title'](https://github.com/mattermost/mattermost-webapp/pull/2689/files#diff-1683b148a56ec2faf3b5c5e35af1a8efR109).  Not sure if there are similar case of this, maybe worth checking.

Reply:
Yes, I should double check those!